microcosm_pubsub/handlers.py

```diff
@@ -64,7 +64,9 @@ def __call__(self, message):
 
     def on_call(self, message, uri):
         self.logger.debug(
-            "Starting {handler}",
+            "Starting {handler}".format(
```

Comment:
Using a string format here is redundant. The console logger will do the formatting for you and in loggly the format string will show up as is (so you can search by it) and the values in extra will be added to the Json.